__DEFINES/setup.dm

```diff
@@ -311,35 +311,36 @@ var/MAX_EXPLOSION_RANGE = 32
 
 // bitflags for mob parts
 
-#define HEAD			1		//specifically the top of the head- imagine it as the scalp.
-#define EYES			2048
-#define MOUTH			4096
-#define EARS			8192
-
-#define UPPER_TORSO		2
-#define LOWER_TORSO		4
-#define LEG_LEFT		8
-#define LEG_RIGHT		16
-#define FOOT_LEFT		32
-#define FOOT_RIGHT		64
-#define ARM_LEFT		128
-#define ARM_RIGHT		256
-#define HAND_LEFT		512
-#define HAND_RIGHT		1024
+#define HEAD			(1<<0)		//specifically the top of the head- imagine it as the scalp.
```

Comment:
This is Hina. Hina tried previously to change these bitflags to shifts. BYOND does not optimize this away, even though it may look cleaner to you, it's just confusing to any newbie coders.

Reply:
🤨 raising my eyebrow rn

Reply:
Ain't no way

Reply:
The "general code cleanup" should just be its own PR

Reply:
this is way more readable than raw powers of 2

Reply:
Change it across the entire codebase in a fresh PR then

Reply:
the entire codebase doesnt have to be consistent, i touched these flags that's why i made it in a consistent format here

Reply:
There is merit in discussing format changes like these, that's why this belongs in a new PR. By sneaking these format changes in here you open this PR for vox tail wagging up for discussion about your format changes. There is a reason atomicity is important